docs/zephyr-integration.md

```diff
@@ -0,0 +1,454 @@
+- **Comprehensive API Coverage**: Full access to Zephyr Cloud functionality through standardized MCP tools
+
+## Setup
+
+### Environment Variables
+
+The Zephyr integration requires the following environment variables:
+
+#### Required
+- `ZEPHYR_ACCESS_TOKEN` (required): JWT access token for Zephyr Cloud API authentication
+  - Obtain from Zephyr Cloud account settings
+  - Format: Standard JWT token (e.g., `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...`)
+
+#### Optional
+- `ZEPHYR_PROJECT_KEY` (optional): Default project scope for operations
+  - Format: Uppercase project key (e.g., `PROJ`, `DEV`, `TEST`)
+  - When specified, operations will default to this project unless overridden
+
+- `ZEPHYR_BASE_URL` (optional): Custom API endpoint for enterprise deployments
+  - Default: `https://api.zephyrscale.smartbear.com/v2`
+  - Format: Full URL including protocol and version
+
+### Configuration Examples
+
+#### Basic Configuration
+```bash
+# Minimum required setup
+export ZEPHYR_ACCESS_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+
+# Start the MCP server
+npx @smartbear/mcp-server
+```
+
+#### Project-Scoped Configuration
+```bash
+# Configuration with default project scope
+export ZEPHYR_ACCESS_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+export ZEPHYR_PROJECT_KEY="MYPROJECT"
+
+# Start the MCP server
+npx @smartbear/mcp-server
+```
+
+#### Enterprise Configuration
+```bash
+# Configuration for enterprise/custom deployment
+export ZEPHYR_ACCESS_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+export ZEPHYR_PROJECT_KEY="CORP"
+export ZEPHYR_BASE_URL="https://zephyr.company.com/api/v2"
+
+# Start the MCP server
+npx @smartbear/mcp-server
+```
+
+### Validation Steps
+
+1. **Token Validation**: Verify your access token is valid and has appropriate permissions
+2. **Project Access**: Ensure the token has access to the specified project(s)
+3. **API Connectivity**: Test connection to the Zephyr Cloud API endpoint
+
+## Available Tools
+
+### Issue Coverage
+
+Tools for analyzing test coverage of JIRA issues and requirements.
+
+#### Get Issue Test Coverage
+- **Purpose**: Retrieve test cases that provide coverage for specific issues
+- **Use Cases**:
+  - Check which test cases cover a specific user story or bug fix
+  - Identify gaps in test coverage for requirements
+  - Generate coverage reports for stakeholders
+- **Parameters**:
+  - `issueKey` (required): JIRA issue key in format PROJECT-123
+  - `projectKey` (optional): Project key to scope the search
+- **Example**:
+  ```json
+  {
+    "issueKey": "PROJ-123",
+    "projectKey": "PROJ"
+  }
+  ```
+
+### Test Case Management
+
+Comprehensive CRUD operations for test case lifecycle management.
+
+#### Create Test Case
+- **Purpose**: Create new test cases with comprehensive metadata
+- **Parameters**:
+  - `name` (required): Test case name
+  - `projectKey` (required): Target project
+  - `folderId` (optional): Organization folder
+  - `priority` (optional): Priority level
```

Comment:
maybe we should inform that if not specified the default values will be set for priority and status